public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+openapi: 3.0.0
+info:
+  title: CTC Guarantee Balance
+  contact: {}
+  version: '2.0'
+servers:
+  - url: https://test-api.service.hmrc.gov.uk/customs/guarantees
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk/customs/guarantees
+    description: Production
+    variables: {}
+paths:
+  /customs/guarantees/:grn/balance:
+    post:
+      tags:
+        - customs
+      summary: Get a Balance
+      description: Used to get the balance associated with the supplied Guarantee Reference Number (GRN)
+      operationId: GetaBalance
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: grn
+          schema:
+            type: string
+            pattern: '^[0-9]{2}[A-Z]{2}[A-Z0-9]{12}[0-9]([A-Z][0-9]{6})?$'
+          required: true
+          description: Guarantee Reference Number of the guarantee to get the balance for
+      requestBody:
+        description: ''
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/CustomsGuaranteesBalancesrequest'
+                - example:
+                    accessCode: ABC1
+            example:
+              accessCode: ABC1
+        required: true
+      responses:
+        '200':
+          description: 'A successful balance request response'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/GetBalanceOk'
+                  - example:
+                      balance: 12345678.90
+              example:
+                _links:
+                  self:
+                    href: /customs/guarantees/23GB0123456789AB2/balance
+                balance: 12345678.90
+        '400':
+          description: 'Bad Request'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/BadRequest'
+                  - description: The GRN or supplied Json payload were invalid
```

Comment:
`Json` should be` JSON`.

Grammatically, the sentence is less than perfect because "or" means using "was". Use of and/or is a consideration but that would mean using were/was, which is undesirable.

Suggestion:
The supplied GRN or JSON payload was invalid.